packages/shared/comms/peers.ts

```diff
@@ -64,8 +64,8 @@ export function setupPeer(address: string): PeerInformation {
   if (!peerMap.has(ethereumAddress)) {
     const peer: PeerInformation = {
       ethereumAddress,
-      lastPositionUpdate: 0,
-      lastProfileUpdate: 0,
+      lastPositionIndex: 0,
+      lastProfileVersion: -1,
```

Comment:
this is uint in the protocol
```suggestion
      lastProfileVersion: 0,
```

Reply:
yes, but this way a profile message with version zero triggers an update, we are not sending this